cita-chain/src/main.rs

```diff
@@ -73,13 +73,14 @@ extern crate libproto;
 extern crate cita_logger as logger;
 #[macro_use]
 extern crate util;
-extern crate db as cita_db;
+
+extern crate cita_database as cita_db;
```

Comment:
You don't need to use `extern` in Edition 2018?

Reply:
How the use `as` without `extern`?

Reply:
Have you try:
```rust
use db as cita_db
```
?

Reply:
Wait a minute.